src/test/java/com/coherent/finalTask/PageBaseClass.java

```diff
@@ -0,0 +1,21 @@
+package com.coherent.finalTask;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+import static com.coherent.finalTask.TestConstants.*;
+
+public class PageBaseClass {
+    protected static WebDriver driver;
+    protected static TestConstants testConstants = new TestConstants();
```

Comment:
TestConstants has public static variables that accessible without TestConstants instance creation